uaclient/cli.py

```diff
@@ -349,11 +361,14 @@ def print_version(_args=None, _cfg=None):
 
 @assert_attached_root
 def action_refresh(args, cfg):
-    if contract.request_updated_contract(cfg):
-        print(ua_status.MESSAGE_REFRESH_SUCCESS)
-        logging.debug(ua_status.MESSAGE_REFRESH_SUCCESS)
-        return 0
-    raise exceptions.UserFacingError(ua_status.MESSAGE_REFRESH_FAILURE)
+    try:
+        contract.request_updated_contract(cfg)
+    except (util.UrlError, exceptions.UserFacingError) as exc:
```

Comment:
This feels like we're discarding some potentially relevant user-facing output by not using the `UserFacingError` we catch. Is that something we could improve?

Reply:
This looks unaddressed, I think?